src/data/inebriety.txt

```diff
@@ -458,7 +458,7 @@ moreltini	1	3	awesome	3-5	0	0	10-20	40 Truffle Tango (+50 Item Drop), MARTINI
 Morlock's Mark Bourbon	4	4	awesome	10-15	10-28	10-28	10-28
 morning dew	1	1	awesome	4	4-6	0	0	40 Granolarrrgh (+20 Stench Damage & Stench Spell Damage)
 Morto Moreto	1	5	EPIC	4-6	0	0	0	100 Majorly Poisoned, WINE
-mostly-empty bottle of cookie wine	1	1	EPIC	5-6	30-40	30-40	30-40	WINE
+mostly-empty bottle of cookie wine	1	1	decent	0	0	0	0	Unspaded WINE
```

Comment:
```suggestion
mostly-empty bottle of cookie wine	1	1	decent	1-2	1-5	1-5	1-5	Unspaded WINE
```
Volc added it to the wiki, and I trust the data.

Reply:
Done